cmd/edge-orchestration/capi/main.go

```diff
@@ -250,6 +251,7 @@ func OrchestrationInit() C.int {
 	return 0
 }
 
+// OrchestrationRequestService performs request from service applications which uses orchestration service
 //export OrchestrationRequestService
```

Comment:
How about removing this line?

Reply:
We got a [warnings](https://goreportcard.com/report/github.com/lf-edge/edge-home-orchestration-go#golint) from the Go Report Card